src/script.ts

```diff
@@ -14,38 +15,53 @@ import {
 
 async function main() {
   const client = new MendixPlatformClient();
-
-  const app = client.getApp(input.AppId);
-  const repo = app.getRepository();
-  const workingCopy = await app.createTemporaryWorkingCopy(input.BranchName);
+  const app = client.getApp(input.AppID);
+  //const repo = app.getRepository();
+  //console.log(await repo.getBranches());
+  const workingCopy = await app.createTemporaryWorkingCopy(input.BranchLine);
   const model = await workingCopy.openModel();
-  const domainModel = await getOrCreateDomainModel(model);
 
-  const objectsFolder = getOrCreateFolder(
-    domainModel.containerAsModule,
-    "objects"
-  );
 
-  let x = 0;
-  let y = 0;
-  for (var ent of input.Entities) {
-    const newEnt = getOrCreateEntity(domainModel, ent.Name, x, y);
-    x += 100;
-    y += 100;
+  for (var mod of input.Modules) {
```

Comment:
Instead of let, use const here. The difference is that if you want to be able to change the iterator inside of the loop, you use let; if no, you use const.